src/Components/CardList.tsx

```diff
@@ -46,21 +46,31 @@ export const CardList = ({ services }: CardListProps): JSX.Element => {
               if (ageNumber === -1) return true
 
               // There is an age band
-              if (
-                item.age?.minAge &&
-                item.age?.maxAge &&
-                item.age.minAge <= ageNumber &&
-                ageNumber <= item.age.maxAge
-              )
-                return true
+              if (item.age?.minAge && item.age?.maxAge) {
```

Comment:
Somehow this feels unwieldy. I'm guessing it is a consequence of how ages are assigned to the content?